package.json

```diff
@@ -108,7 +108,7 @@
     "check-licenses": "wp-scripts check-licenses --prod --gpl2",
     "build": "wp-scripts build",
     "dev": "wp-scripts start",
-    "env": "wp-env",
+    "wp-env": "wp-env",
```

Comment:
So the tests pass. It worked locally before, but it looks like it has to be `wp-env` in CircleCI.